### ISSUE TITLE
Missing guide from wiki page.

### ISSUE THREAD

**Issue by ragingenigma** [SubnauticaNitrox/Nitrox#294]
The wiki lacks enough information to complete the compilation or usage of the server mod.

There was some information posted on the wiki last week.  It seems to have been purposely removed.

Can we please have this information put back on the wiki?

**Comment by ragingenigma** [comment]
I strongly disagree.  I consider myself a 'starting' developer and the _so called_ information does not give _useful_ information on implementation of the mod.

It also seems the statement a deception.  This project does not discourage.  The project is refusing to provide _proper_ information so that other would be developers can help.  Github is about open source projects where different people can contribute to the good of the project.  It is obvious you do not want contribution.

**Comment by kylinator25** [comment]
Well, what part of the wiki don't you understand? If we can work out the problem perhaps we can reach a middle ground. (Keep in mind that by developers this mod generally refers to those with some good C# experience under their belt)

**Comment by ahk1221** [comment]
If you do not understand how to use git or how to use dnSpy (which is what is required for making this mod work) then you are not fit for using this mod. The videos you have linked contain misinformation, which is the main reason they have been taken down.

**Comment by ragingenigma** [comment]
I am a 'starting' developer.  While that gives me ability to write code, the injecting step is unclear to me.  How do you determine a suitable method?  And locating the GameInput.Awake() method has been difficult.

@ahk1221 - So you are telling me that I have to know how _every_ program works of other developers?  I do not know what videos you are talking about.  I came across this mod because a friend suggested it.  I do not waste my time watching videos.

**Comment by Measurity** [comment]
@ragingenigma 

I welcome you to our discord to discuss this further because GitHub isn't a suitable place to have a back-and-forth discussion. https://discord.gg/PUtT8P

**Comment by ragingenigma** [comment]
I searched for the Game.Awake method as it was written.  dnSpy is not set to search for things in that manner. I am new to the program and have not learned it's conventions yet.  This is why I wanted clearer instructions. I will muddle through.

@Measurity 
Thank you for the invite, but I am not comfortable communicating in that manner. {personal limitations}
If there is a forum, or other means to discuss this, I can contribute in that manner.
